firmware/thruna/BMS/Test/Src/Test_Faults.cpp

```diff
@@ -566,14 +566,14 @@ TEST_F(BmsFaultTest, check_state_transition_fault_state_precharge_fault)
         is_air_negative_closed_fake.return_val = true;
         is_charger_connected_fake.return_val   = false;
         App_CanRx_Debug_StartCharging_Update(false);
-        LetTimePass(state_machine, 10U);
+        LetTimePass(state_machine, 250);
```

Comment:
So Inverter state requires an extra 200ms, have you tried reducing this closer to 210-220 to see if it still passes?

Also whatever number you settle on, add a 'U' at the end to mark it as unsigned